Override ossec_rules.xml to disable netstat check

conorbranagan opened the pull request:
This includes a [netstat check](https://github.com/ossec/ossec-hids/blob/master/etc/rules/ossec_rules.xml#L149) that uses a good chunk of CPU on hosts that have many active connections. We were seeing spikes up to 60-80% CPU which hurts the main service running on machines.

alq666 commented:
@conorbranagan instead of this could you have [this file](https://github.com/ossec/ossec-hids/blob/master/etc/rules/ossec_rules.xml#L149) as a `cookbook_file` with the netstat bit commented out?

alq666 commented on templates/default/ossec.conf.erb:
```diff
@@ -94,7 +94,6 @@
     <include>vmware_rules.xml</include>
     <include>ms_dhcp_rules.xml</include>
     <include>asterisk_rules.xml</include>
-    <include>ossec_rules.xml</include>
```
A bit heavy-handed to remove it altogether.

dougdep commented:
beat me to it @alq666 would rather just comment that line out. I assume those rules get pulled down during install?

devolur commented:
I am fine with @conorbranagan doing whatever needs to be done to fix this issue, including removing the rule.

conorbranagan committed:
Override ossec_rules.xml with cookbook file.

Our override disables the netstat check that doesn't scale well.

New recipe ossec::rules supports overriding or adding new rules to the standard
deployment.

alq666 commented:
@conorbranagan 👍
